src/main/java/com/newrelic/experts/client/model/JenkinsMasterEvent.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.newrelic.experts.client.model;
+
+/**
+ * A JenkinsMasterEvent.
+ * <p>
+ * Describes monitorable events on Jenkins master.
+ * </p>
+ * @author Dan Alvizu (alvizu@gmail.com)
+ */
+public class JenkinsMasterEvent
```

Comment:
@dalvizu Having a sub-class here is more elegant but also seems unnecessary. The current pattern we follow is simply creating a new `Event` in the `record*` method and setting the attributes prior to being recorded.  I can see the need to have specific `*Event` classes for polymorphic reasons and if the event had a number of attributes that made the constructor require a larger number of arguments.  Where we would only need 2 here, I would prefer to just stick with the current way we do this for the other events.  If we _do_ go with separate events, I'd prefer to do it consistently so that _each_ of the events produced has it's own sub-class.

Reply:
I'll remove this class. My main reason was for having a place to encapsulate details about NewRelic (needing to pass integers as floats) and document uncommon Jenkins features (what 'quiet down mode' is)

I still think that's valuable but I definitely see the value in consistency.